BlockParty.py

```diff
@@ -374,6 +386,51 @@ def put_figure(self):
                 if i + self.py < self.bh and self.figure[i][j] != 0:
                     self.glass[i + self.py][j + self.px] = self.figure[i][j]
 
+    def generate_ghost_py(self, figure):
+        ghost_py = self.py - 1
+        while self.figure_fits(ghost_py, figure):
+            ghost_py -= 1
+        return ghost_py + 1
+
+    def get_ghost_color(self, color, mix_intensity=0.5):
+        r = color.red
+        g = color.green
+        b = color.blue
+        r_int = int(r * 255)
+        g_int = int(g * 255)
+        b_int = int(b * 255)
```

Comment:
No need for the r,g,b variables when you can do same in (r,g,b)_int, also see if this can be made simpler with `Gdk.RGBA`.